app/controllers/root_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RootController < ApplicationController
+  def index
+    respond_to do |format|
+      format.html # Render the HTML template
+      format.json { render json: { message: 'JSON response' } }
+    end
+  end
+end
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍